dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/Sqrt.mo

```diff
@@ -0,0 +1,78 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2023, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+block Sqrt "Outputs the square root of the input"
```

Comment:
The fact that this Sqrt bloc uses a NoEvent should be clear in its name

Reply:
We can rename it SqrtNoEvent and add a description "This sqrt block, derived from the sqrt Modelica block, uses noEvent function to avoid calculating negative u values"